### ISSUE TITLE
Button to apply object name to mesh name

### ISSUE THREAD

**Issue by Rodg88** [taniwha/io_object_mu#42]
just a general blender tool suggestion that could be useful, but an option to copy the selected object's name to the object's mesh would be useful.
Not sure what it'd do with multi-user meshes, maybe if only one object is selected it could work, but if multiple objects are selected it'd skip the multi-user meshes.

Could maybe be a separate addon, as it would be useful beyond just KSP modelling

**Comment by StoneBlue** [comment]
Yes, please... I find myself having to rename tons of object names quite often... anything to mae it easier would be nice. 
Idk if its just me, but copy/pasta doesnt wor in the Outliner for object names.

Also, sorta related, mebbe...
I also notice copy/paste doesnt wor in the texture names fields in your Mu material properties panel... vOv

**Comment by popos123** [comment]
yeah, it will be nice, maybe is a plugin for a blender to do that, in meantime i have piece of code in python, eg.:
`    # Rename remaining objects to "DepthMask"
    for obj in bpy.context.scene.objects:
        if obj.type in {'MESH', 'EMPTY'}:
            obj.name = "DepthMask"`
